src/components/FocusZone/FocusZone.tsx

```diff
@@ -114,10 +114,40 @@ export class FocusZone extends React.Component<IFocusZoneProps, {}> implements I
     } else {
       const firstChild = this.refs.root.firstChild as HTMLElement;
 
-      return this._focusElement(getNextElement(this.refs.root, firstChild, true));
+      return this.focusElement(getNextElement(this.refs.root, firstChild, true));
     }
   }
 
+  /** Sets focus to a specific element within the zone. */
```

Comment:
nit: Documentation of methods should always use the expanded JSDoc form, only use the single-line version for things like enum properties.